src/app/wizard/pages/intro/intro.component.html

```diff
@@ -23,8 +23,8 @@ <h1>Supported Runtimes</h1>
       <div class="col-sm-3 runtime">
```

Comment:
we are using bootstrap grid system there are 12 columns now there are 15. see https://getbootstrap.com/docs/3.3/css/#grid-options

Reply:
what do you suggest? `class="col-sm-2"` for all?

Reply:
better to split it into 2 rows